src/nvim/window.c

```diff
@@ -3787,6 +3787,7 @@ void free_tabpage(tabpage_T *tp)
 /// @return Was the new tabpage created successfully? FAIL or OK.
 int win_new_tabpage(int after, char_u *filename)
 {
+  tabpage_T   *old_curtab = curtab;
   tabpage_T   *tp = curtab;
```

Comment:
lets move this variable inside the else block line 3814 and use `old_curtab` for the other usages, to be consequent.

Reply:
Thank you for review, fixed it!